### ISSUE TITLE
Docker not auto-started on Centos after CC install script

### ISSUE THREAD

**Issue by grahamwhaley** [01org/cc-oci-runtime#818]
After following #799 in a Centos VM running under QEMU, after a reboot of the VM it looks like docker is not automatically started. A `sudo systemctl start docker` fixes it.
We should probably look at enabling docker by default.

**Comment by dlespiau** [comment]
I can see argued that it may be beyond the scope of what an installer script should do and up to the system administrator.

**Comment by grahamwhaley** [comment]
Agreed - I can see this as 'borderline'. At a minimum we may wish to mention it in the install instructions docs, as a reboot results in a non-functional docker, which our script installed.
I presume the standard Centos docker install does not enable itself by default, and it is not something we have done or disabled in the script for instance - in which case I'd be happy with us just documenting the state, and ideally a 'fix' as well.
